sensor_msgs/PointCloud2 cloud
float32 gap
float32[] sensor_position
float32[] sensor_z
float32[] sensor_y
float32[] sensor_x
bool first
bool last